### PR TITLE
move packaging string to psd1 file

### PR DESCRIPTION
**Pull request by TravisEz13**
move here strings in packaging module to psd1 file for easier maintenance of strings and code

### CODE REVIEW

**Review by lzybkr — COMMENTED** [review]
We should only use locale specific resources for strings that might require localization.

**Comment by TravisEz13** [comment]
@lzybkr I moved the psd1 out of a local specific folder, does that address your concern?

**Comment by lzybkr** [comment]
You still use `Import-LocalizedData` - remove that and my concerns are addressed.

File: tools/packaging/packaging.psm1 [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+        [string]$Path
+    )
+
+    $ErrorActionPreference = 'Stop'
+
+    $psd1 = Get-Content -Raw -Path $Path
+    [ScriptBlock]$scriptBlock = [ScriptBlock]::Create($psd1)
+    [string[]] $allowedCommands = 'ConvertFrom-StringData'
+    $scriptBlock.CheckRestrictedLanguage($allowedCommands,$null,$false)
+    return $scriptBlock.InvokeReturnAsIs()
```
Comment (JamesWTruher) [inline tools/packaging/packaging.psm1]: you could add `[pscustomobject]` here

**Review by JamesWTruher — APPROVED** [review]

**Comment by TravisEz13** [comment]
I filed #4337 for a fix to PowerShell, which won't help here as this script needs to run down level as well.
@lzybkr I added `Import-Data` which does almost the same thing as `Import-LocalizedData` but with out the localized file look-up

File: tools/packaging/packaging.psm1 (hunk 1 above)
Comment (lzybkr) [inline tools/packaging/packaging.psm1]: Why not just use the data statement in a ps1 file and run it?

Comment (TravisEz13) [inline tools/packaging/packaging.psm1]: I'd like to have a file that is restricted to data. In a similar pattern as `Import-LocalizedData`.

Comment (lzybkr) [inline tools/packaging/packaging.psm1]: You can use `Import-PowerShellDataFile` - it's more restrictive than it needs to be for pure data, but I believe already covers your use case.

**Comment by daxian-dbw** [comment]
All comments have been addressed, so I'm about to merge this PR.
@TravisEz13 Since `Import-PowerShellDataFile` is the command to import a .psd1 file, I think #4337 probably should be closed. Please re-open the issue if you think otherwise.

**Pull request merged**